samples/bluetooth/broadcast_audio_sink/boards/nrf52833dk_nrf52833.overlay

```diff
@@ -0,0 +1,15 @@
+zephyr_udc0: &usbd {
+	compatible = "nordic,nrf-usbd";
+	status = "okay";
+
+	hs_0: hs_0 {
+		compatible = "usb-audio-hs";
+		mic-feature-mute;
+		mic-channel-l;
+		mic-channel-r;
+
+		hp-feature-mute;
+		hp-channel-l;
+		hp-channel-r;
+	};
+};
```

Comment:
Is there a guideline for when these should be added to the overlay files, and when they should be added to the board definitions? 

e.g. the 5340 ADK has hs_0 in the board definition.

Reply:
These normal DKs with USB support in SoC do not have to have USB Audio enabled, consuming code space for the USB stack. The features being software defined will depend on the applications, hence best defined in the applications boards overlay.